.github/workflows/push-mls.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy Nodes
+on:
+  push:
+    branches:
+      - mls
```

Comment:
Is the plan to keep this on the `mls` branch for now or merge into `main`? Noticed that the PR is opened against the `mls` branch too

Reply:
Plan is to use the MLS branch for at least this week, and work out all the kinks with the schema. Then merge to main and set the appropriate server flags.